pom.xml

```diff
@@ -473,6 +473,12 @@
       <groupId>org.slf4j</groupId>
       <artifactId>slf4j-api</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.xerial.snappy</groupId>
+      <artifactId>snappy-java</artifactId>
+      <scope>runtime</scope>
```

Comment:
should we make it test only since this is only for testing?

Reply:
It is not testing only, this is transitive dependency of hadoop-common